### DISCUSSION TITLE
creating unique style

### DISCUSSION THREAD

**Discussion by MRDHJJJER** [lllyasviel/Fooocus#2273] [General]
can I use my own dataset to create unique style and then use it to generate similar images to it? if it can be done how do I do it?

**Comment by poor7** [comment]
Certainly. Look at how [styles](https://github.com/lllyasviel/Fooocus/tree/main/sdxl_styles) are arranged (it's just a certain set of prompts) and do something similar.

**Comment by mashb1t** [comment]
You can also check out kohya_ss, a tool for LoRA and model generation. This supports training with your own data set of images.
